app/jobs/organization_event_job.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Documentation: https://developer.github.com/v3/activity/events/types/#organizationevent
+class OrganizationEventJob < ApplicationJob
+  queue_as :github_event
+
+  def perform(payload_body)
+    return true unless payload_body.dig("action") == "member_removed"
+
+    github_user_id = payload_body.dig("membership", "user", "id")
+    github_organization_id = payload_body.dig("organization", "id")
+    organization = Organization.find_by(github_id: github_organization_id)
+    user = organization.users.find_by(uid: github_user_id)
+
+    return true unless user
+
+    organization.users.delete(user)
+    organization.save
```

Comment:
Since this is a `has_and_belongs_to_many` relationship, the organization doesn't need to be saved here